kura/org.eclipse.kura.linux.watchdog/src/main/java/org/eclipse/kura/linux/watchdog/WatchdogServiceImpl.java

```diff
@@ -72,7 +74,7 @@ public void updated(Map<String, Object> properties) {
         if (this.options.isEnabled()) {
             this.rebootCauseWriter = new RebootCauseFileWriter(this.options.getRebootCauseFilePath());
 
-            try (PrintWriter wdWriter = new PrintWriter(this.options.getWatchdogDevice());) {
+            try (PrintWriter wdWriter = new PrintWriter(WD_FILE);) {
```

Comment:
Just a clarification: the WD_FILE location is now somewhat hard coded in the implementation. But it is still modifiable in the service preferences.
There is still a reference to the `getWatchdogDevice` in the private method `writeWatchdogDevice`. Could this take to some sort of inconsistencies? What happens if the user changes that value in the service preferences?